arch/arm/src/stm32l5/hardware/stm32l562xx_pinmap.h

```diff
@@ -0,0 +1,745 @@
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.  See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ *
+ ****************************************************************************/
+
+#ifndef __ARCH_ARM_SRC_STM32L5_HARDWARE_STM32L562XX_PINMAP_H
+#define __ARCH_ARM_SRC_STM32L5_HARDWARE_STM32L562XX_PINMAP_H
+
+/****************************************************************************
+ * Included Files
+ ****************************************************************************/
+
+#include <nuttx/config.h>
+
+/****************************************************************************
+ * Pre-processor Definitions
+ ****************************************************************************/
+
+/* Alternate Pin Functions.  All members of the STM32L552xx and STM32L562xx
+ * families share the same pin multiplexing (although they may differ in the
+ * pins physically available).  See DS12736, Table 22 "Alternate Function AF0
+ * to AF7" and Table 23 "Alternate Function AF8 to AF15".
+ *
+ * Alternative pin selections are provided with a numeric suffix like _1, _2,
+ * etc.  Drivers, however, will use the pin selection without the numeric
+ * suffix.  Additional definitions are required in the board.h file.  For
+ * example, if FDCAN1_RX connects vis PA11 on some board, then the following
+ * definitions should appear inthe board.h header file for that board:
+ *
+ * #define GPIO_FDCAN1_RX GPIO_FDCAN1_RX_1
+ *
+ * The driver will then automatically configure PA11 as the FDCAN1 RX pin.
+ */
+
+/* ADC */
+
+#define GPIO_ADC12_IN1         (GPIO_ANALOG|GPIO_PORTC|GPIO_PIN0)
+#define GPIO_ADC12_IN2         (GPIO_ANALOG|GPIO_PORTC|GPIO_PIN1)
+#define GPIO_ADC12_IN3         (GPIO_ANALOG|GPIO_PORTC|GPIO_PIN2)
+#define GPIO_ADC12_IN4         (GPIO_ANALOG|GPIO_PORTC|GPIO_PIN3)
+#define GPIO_ADC12_IN5         (GPIO_ANALOG|GPIO_PORTA|GPIO_PIN0)
+#define GPIO_ADC12_IN6         (GPIO_ANALOG|GPIO_PORTA|GPIO_PIN1)
+#define GPIO_ADC12_IN7         (GPIO_ANALOG|GPIO_PORTA|GPIO_PIN2)
+#define GPIO_ADC12_IN8         (GPIO_ANALOG|GPIO_PORTA|GPIO_PIN3)
+#define GPIO_ADC12_IN9         (GPIO_ANALOG|GPIO_PORTA|GPIO_PIN4)
+#define GPIO_ADC12_IN10        (GPIO_ANALOG|GPIO_PORTA|GPIO_PIN5)
+#define GPIO_ADC12_IN11        (GPIO_ANALOG|GPIO_PORTA|GPIO_PIN6)
+#define GPIO_ADC12_IN12        (GPIO_ANALOG|GPIO_PORTA|GPIO_PIN7)
+#define GPIO_ADC12_IN13        (GPIO_ANALOG|GPIO_PORTC|GPIO_PIN4)
+#define GPIO_ADC12_IN14        (GPIO_ANALOG|GPIO_PORTC|GPIO_PIN5)
+#define GPIO_ADC12_IN15        (GPIO_ANALOG|GPIO_PORTB|GPIO_PIN0)
+#define GPIO_ADC12_IN16        (GPIO_ANALOG|GPIO_PORTB|GPIO_PIN1)
+
+/* FDCAN */
+
+#define GPIO_FDCAN1_RX_1       (GPIO_ALT|GPIO_AF9 |GPIO_PORTA|GPIO_PIN11)
```

Comment:
Formatting is not consistent

Reply:
You mean because of the blank after GPIO_AF9, right? Ok, will remove all blanks. Thanks.

Reply:
yes

Reply:
Please have a look at https://github.com/apache/incubator-nuttx/pull/2974/commits/233c38d9adce7a28767bb2edfabb772df0d0fe5b